rainforest/tests.go

```diff
@@ -197,6 +197,23 @@ func (c *Client) GetRFMLIDs() (TestIDMappings, error) {
 	return testResp, nil
 }
 
+// GetTest gets a test from RF specified by the given test ID
+func (c *Client) GetTest(testID int) (*RFTest, error) {
+	req, err := c.NewRequest("GET", "tests/"+strconv.Itoa(testID), nil)
```

Comment:
I only did it this way because I thought it was more efficient. I'll take a look at our string concatenation in a diff PR since it's all over the project.